### DISCUSSION TITLE
Code view for HTML

### DISCUSSION THREAD

**Discussion by kumaheiyama** [sibiraj-s/ngx-editor#502] [Ideas]
It would be good to be able to see the HTML markup. preferably to be able to switch between WYSIWYG and markup mode if you need to change something specific.

**Comment by jmullings** [comment]
import { ToolbarDropdown, Toolbar, Editor } from "ngx-editor";

  convertToHTML(content: any): string {
    const serializer = DOMSerializer.fromSchema(this.editor.schema);
    const fragment = serializer.serializeFragment(content);
    const div = document.createElement("div");
    div.appendChild(fragment);
    return div.innerHTML;
  }
